app/services/payment_providers/adyen_service.rb

```diff
@@ -2,21 +2,95 @@
 
 module PaymentProviders
   class AdyenService < BaseService
+    WEBHOOKS_EVENTS = [
+      'AUTHORISATION', 'REFUND', 'REFUND_FAILED'
+    ].freeze
+
     def create_or_update(**args)
       adyen_provider = PaymentProviders::AdyenProvider.find_or_initialize_by(
-        organization_id: args[:organization].id,
+        organization_id: args[:organization].id
```

Comment:
Style/TrailingCommaInArguments: Put a comma after the last parameter of a multiline method call. (https://rubystyle.guide#no-trailing-params-comma)